build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/AnalyticsUtils.java

```diff
@@ -6,7 +6,7 @@
 import io.quarkus.test.services.quarkus.model.QuarkusProperties;
 
 public class AnalyticsUtils {
-    private static final String QUARKUS_SNAPSHOT_VERSION = "3.26.999-SNAPSHOT";
+    private static final String QUARKUS_SNAPSHOT_VERSION = "3.27.999-SNAPSHOT";
     // Quarkus reports extensions' versions in the analytics payload as full artifact version.
     // So for 999-SNAPSHOT, it can look like e.g. 999-20230718.203351-1091
     private static final String QUARKUS_EXTENSION_SNAPSHOT_VERSION_REGEX = "3\\.26\\.999-.*";
```

Comment:
```suggestion
    private static final String QUARKUS_EXTENSION_SNAPSHOT_VERSION_REGEX = "3\\.27\\.999-.*";
```

The regex here should be also updated